### PR TITLE
Fix ufgServer options and consider subConfig

### PR DESCRIPTION
**Pull request by SophieDePaula**
This PR fix a few things:

1) [This trello card](https://trello.com/c/qtepTmGL/1832-canvastorybook3321layoutbreakpoints-issue) - send ufgServer rather than ufg to makeUFGClient
2) renderStories integration test
3) I didn't consider layoutBreakpoints and waitBeforeCapture if sent throw sub config

TODO:

- [x] Write e2e test for layoutBreakpoints story config.
- [x] Write e2e test for waitBeforeCapture story config
- [x] pending changes

### CODE REVIEW

File: pending-changes.yaml [hunk 1]
```diff
@@ -66,7 +66,8 @@ Apollo SDK:
   feature:
     - 
   bug-fix:
-    - 
+    - Make sure to pass LayoutBreakpoints and WaitBeforeCapture from story config to dom snapshot
```
Comment (SophieDePaula) [inline pending-changes.yaml:69]: @benbabayoff8

File: js/packages/eyes-storybook/src/getStoryData.js [hunk 2]
```diff
@@ -58,7 +65,7 @@ function makeGetStoryData({logger, takeDomSnapshots, waitBeforeCapture, reloadPa
     const domSnapshotsPromise = takeDomSnapshots({
       page,
       renderers,
-      layoutBreakpoints: eyesParameters ? eyesParameters.layoutBreakpoints : undefined,
+      layoutBreakpoints: eyesParameters ? eyesParameters.layoutBreakpoints : layoutBreakpoints,
```
Comment (kyrylodolynskyi) [inline js/packages/eyes-storybook/src/getStoryData.js:68]: This is covered here - https://github.com/applitools/eyes.sdk.javascript1/pull/1490

**Review by kyrylodolynskyi — APPROVED** [review]

File: pending-changes.yaml [hunk 3]
```diff
@@ -66,7 +66,8 @@ Apollo SDK:
...
+    - Mapp UFG config correctly
```
Comment (tourdedave) [inline pending-changes.yaml:70]: Typo?

**Review by tourdedave — APPROVED** [review]
LGTM, aside from the typo in pending changes (but I'm not sure if that entry is still relevant).

**Pull request merged**